apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx

```diff
@@ -123,7 +123,7 @@ const Delegations = (props: DelegationsPropsType) => {
    * Return an introductory panel to delegations if the user hasn't previously delegated.
    */
 
-  if (delegatedAmount.eq(0) && rewardsAmount.eq(0)) {
+  if (delegatedAmount.isEqualTo(0) && rewardsAmount.isEqualTo(0)) {
```

Comment:
I've actually changed them for better readability when going through the code. I can revert if necessary?

Reply:
Nope that's fine :+1: thanks for clarifying.

Reply:
The difference is that .eq() returns a boolean and .isEqualTo() returns a BigNumber. 
We will switch back to .eq() since it is more appropiate in our case.